README.md

```diff
@@ -489,6 +489,31 @@ $ git push heroku master
 If the app is deployed successfully, congratulations! You've finished setting
 up, deploying to, and hosting your bot on Heroku!
 
+## Troubleshooting
+
+Sometimes mistakes happen and like with most programming related problem - the solution is reading logs and documentation. 
+To check your Heroku app logs you can use command `heroku logs --tail` and read what might be cause of your problem.
+
+In case you see following line in your logs:
+
+`Error R10 (Boot timeout) -> Web process failed to bind to $PORT within 60 seconds of launch`
+
+Then you should check if you are running your app correct dyno - It should running it on worker and not on web, 
+you can read about different types of dyons [here](https://devcenter.heroku.com/articles/dynos).
+To find out what kind on dyno your application is running you use can go to 
+[Heroku](https://dashboard.heroku.com/apps)
+website and check under your app Resources tab or using command line `heroku ps`. 
+There should be only one worker dyno - if not you can change it using UI or by following commands: 
+
+```sh
```

Comment:
Can you be more explicit on what each of the commands does? Also, I believe you can combine scaling web and worker commands together; there's no need for 2 separate commands.